python/lsst/ci/hsc/validate.py

```diff
@@ -189,6 +189,14 @@ class SfmValidation(Validation):
     def validateSources(self, dataId):
         catalog = Validation.validateSources(self, dataId)
         self.checkApertureCorrections(catalog)
+        # Check that at least 95% of the stars we used to model the PSF end up classified as stars
+        psfStars = catalog.get("calib_psfUsed")
+        extStars = catalog.get("base_ClassificationExtendedness_value") < 0.5
+        self.assertGreater(
+            "At least 95% of sources used to build the PSF are classified as stars",
+            numpy.logical_and(extStars, psfStars).sum(),
+            0.95*psfStars.sum()
```

Comment:
Based on the logic used in MeasureValidation, 95% should be good enough to make sure that the aperture correction and extendedness are configured reasonably. I've added a small note to the documentation to indicate where the 95% comes from.